src/main/java/course_project/Main.java

```diff
@@ -0,0 +1,13 @@
+package course_project;
+
+import course_project.SeaBattle.controller.GameController;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        GameController gameController = new GameController();
```

Comment:
![image](https://user-images.githubusercontent.com/58872509/134058408-409d05e1-b07b-4cac-b449-b8d18c4575a7.png)
Is it a bug or a feature?)

Reply:
Also, it seems computer hit 2 times, then missed, but the message is only missed 
![image](https://user-images.githubusercontent.com/58872509/134058607-d41154a0-9f84-466e-9004-2fc4b32c4eed.png)

Reply:
The interface is really cool, i like I. Few improvements: print messages how computer shoots, where, and hit or miss. For now it is not informative. And also computer logic shoots randomly, it doesn't want to kill already hit ships. Optional

Reply:
let's consider this a feature :))

Reply:
There's a lot of static methods, it's not recommended. Better work with return values

Reply:
Really nice implementation of SeaBattle with a lot of features and written code. Nice usage of abstractions, Java Core and Java libraries, good mvc model, even if not in web application, abstractions are chosen correctly, code is clean and mostly easy to read. Well-done! Approved.

Reply:
> Also, it seems computer hit 2 times, then missed, but the message is only missed
> ![image](https://user-images.githubusercontent.com/58872509/134058607-d41154a0-9f84-466e-9004-2fc4b32c4eed.png)
maybe the computer immediately hit the ship and got a second move

Reply:
Right, second and third (third is miss), but there's no any notification about this, only changed grid. Quite difficult to follow these leaps

Reply:
Add at least messages that computer hit 4 times, then missed

Reply:
> Add at least messages that computer hit 4 times, then missed

Added